dev-tools/reconfigurator-sp-updater/src/main.rs

```diff
@@ -421,6 +421,14 @@ enum Component {
         #[arg(long, short = 't')]
         expected_transient_boot_preference: Option<RotSlot>,
     },
+    RotBootloader {
+        /// expected version of the active slot
+        #[arg(long, short = 'a')]
+        expected_stage0_version: ArtifactVersion,
+        /// expected version of the inactive slot
```

Comment:
Are "active" and "inactive" right in the comments for these? Should maybe be `stage0` and `stage0 next`?